end-to-end-test/shared/wdio/webdriver-manager.conf.js

```diff
@@ -319,21 +333,17 @@ var config = {
     // }
 };
 
-const doBrowserstack = false;
-
-if (doBrowserstack) {
-    config.capabilities[0]['browserstack.local'] = true;
+if (process.env.TEST_IE11 === 'true') {
+    config.capabilities = config.IECapabilties;
+}
 
+if (process.env.TEST_BROWSERSTACK === 'true') {
     config.services = ['visual-regression', 'browserstack'];
-
     config.browserstackLocal = true;
-
-    config.user = process.env.BROWSERSTACK_USER;
-    config.key = process.env.BROWSERSTACK_KEY;
+    config.user = 'aaronlisman2'; //process.env.BROWSERSTACK_USER;
```

Comment:
ok, i will hide these in circle.  and change the password since this is now in git history